user/kyotocabinet/template.py

```diff
@@ -0,0 +1,14 @@
+pkgname = "kyotocabinet"
+pkgver = "1.2.80"
+pkgrel = 0
+build_style = "configure"
+configure_args = ["--prefix=/usr"]
+hostmakedepends = ["pkgconf"]
+makedepends = ["zlib-ng-compat-devel"]
+pkgdesc = "Another straightforward implementation of DBM"
+maintainer = "metalparade <comer@live.cn>"
+license = "GPL-3.0-or-later"
+url = "https://dbmx.net/kyotocabinet"
+source = f"{url}/pkg/kyotocabinet-{pkgver}.tar.gz"
+sha256 = "4c85d736668d82920bfdbdb92ac3d66b7db1108f09581a769dd9160a02def349"
+options = ["autosplit", "splitstatic"]
```

Comment:
```suggestion
```

these are the default, you don't have to specify these

these packages are also still missing the -devel subpackage and similar, read e.g. main/libass like i linked for what you are supposed to do.

Reply:
Hey, finally got all those templates right, you might want to check them again. Still huge thanks, learned a lot during this whole thing!